test/e2e/angular/font-setting-test-ng.js

```diff
@@ -11,108 +11,222 @@
+        });
+
+        it("should not select italic", function() {
+          expect(element(by.css(".mce-active[aria-label='Italic']")).isPresent()).to.eventually.be.false;
+        });
+
+        it("should set correct font style for preview text", function() {
+          expect(element(by.css(".text")).getCssValue("font-style")).to.eventually.equal("normal");
+        });
+      });
+
+      describe("Underline", function() {
+        it("should show underline", function() {
+          expect(element(by.css(".mce-btn[aria-label='Underline']")).isDisplayed()).to.eventually.be.true;
+        });
+
+        it("should not select underline", function() {
+          expect(element(by.css(".mce-active[aria-label='Underline']")).isPresent()).to.eventually.be.false;
+        });
+
+        it("should set correct text decoration for preview text", function() {
+          expect(element(by.css(".text")).getCssValue("text-decoration")).to.eventually.equal("none");
+        });
+      });
     });
 
-    it("Alignment should be present and dialog closed", function() {
-
-      expect(element(by.css(".btn-alignment .fa-align-left"))
-        .isPresent()).to.eventually.be.true;
-      expect(element(by.css(".alignment div.dropdown-menu")).isDisplayed())
-        .to.eventually.be.false;
-
-    });
-
-    it("Preview should show default values", function () {
-      var elem = element(by.css(".font-picker-text span"));
-      
-      expect(elem.getCssValue("font-family")).to.eventually.equal("Verdana");
-
-      expect(elem.getCssValue("font-size")).to.eventually.equal("20px");
-
-      expect(elem.getCssValue("font-weight")).to.eventually.equal("normal");
-
-      expect(elem.getCssValue("font-style")).to.eventually.equal("normal");
-
-      //expect(elem.getCssValue("text-decoration")).to.eventually.equal("none");
-      expect(elem.getCssValue("text-decoration")).to.eventually.match(/^none/);
-
-      expect(elem.getCssValue("color")).to.eventually.equal("rgba(0, 0, 0, 1)");
-
-      expect(elem.getCssValue("background-color")).to.eventually.equal("rgba(0, 0, 0, 0)");
-
-
-      expect(element(by.css(
-        ".btn-alignment[data-wysihtml5-command-value='left']")).isPresent()).
-        to.eventually.be.true;
-      expect(element(by.css(".font-picker-text")).getCssValue("text-align")).
-        to.eventually.equal("left");
-
-    });
-
-    // Bold
-    it("Should turn bold on when clicked", function () {
-      element(by.css(".bold")).click();
-
-      expect(element(by.css(".bold.active")).isPresent()).
-        to.eventually.be.true;
-      expect(element(by.css(".font-picker-text span")).getCssValue("font-weight")).
-        to.eventually.equal("bold");
+    describe("Updating", function() {
+      describe("Font Family", function() {
+        it("should update font family for preview text", function() {
+          element(by.css(".mce-btn[aria-label='Font Family']")).click();
+          element(by.css("#mceu_21-text")).click();
```

Comment:
@stulees Because the font family and font size selectors use the same CSS classes and attributes, I can't distinguish between them. Only thing I can think of is to apply a custom class to them and then I could just select the first child font. But I don't recall having come across a way of doing that. Have you?